.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,22 @@
-**I'm submitting a ...**
-- [ ] bug report
-- [ ] feature request
+# Template to submit an issue
```

Comment:
@optimistanoop  Only add
  -  [ ] I have read the [contribution guidelines](https://github.com/UdacityMobileWebScholarship/showcase-app/blob/master/.github/CONTRIBUTING.md)